src/main/java/com/getify/minify/Minify.java

```diff
@@ -0,0 +1,103 @@
+//
+//  Minify.java
+//  jsonminifier - Ported from https://github.com/getify/JSON.minify
+//
+//  Created by Bernhard Gass on 8/01/13.
+//  Copyright © 2013 Bernhard Gass. All rights reserved.
+
+package com.getify.minify;
```

Comment:
I don't really do any Java work. The only reason this is a Java project at all is as a language port of the original implementation. I think it's very unlikely that I'll do future Java work, but it's not entirely implausible that some other project might also have a Java port in a similar vein as this one. Even still, I am only the host of that code, it's not code I write or actively maintain.